src/state/zustand/swapperStore/selectors.ts

```diff
@@ -40,6 +40,7 @@ export const selectIsSendMax = (state: SwapperState) => state.isSendMax
 export const selectAmount = (state: SwapperState) => state.amount
 export const selectReceiveAddress = (state: SwapperState) => state.receiveAddress
 export const selectFees = (state: SwapperState) => state.fees
+export const selectProtocolFees = (state: SwapperState) => state.fees?.protocolFees
```

Comment:
I was going to suggest `createSelector` and leveraging `selectFees` but we don't use it anywhere in this module, disregard me (though it should be possible https://www.phind.com/search?cache=12ec4784-3efe-4cd4-8e98-9bbc9bc67e36)